pkg/logs/config/integration_config.go

```diff
@@ -27,7 +27,8 @@ type LogsConfig struct {
 	Port int    // Network
 	Path string // File, Journald
 
-	ExcludePaths []string `mapstructure:"exclude_paths" json:"exclude_paths"` // File
+	ExcludePaths []string `mapstructure:"exclude_paths" json:"exclude_paths"`   // File
+	TailingMode  string   `mapstructure:"start_position" json:"start_position"` // File
```

Comment:
```suggestion
	TailingMode    string `mapstructure:"start_position" json:"start_position"` // File
```

Reply:
Unfortunately the original indentation is the one coming out from go fmt.